### PR TITLE
Introduce user activity page

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: ui/app/com/gu/recipeasy/controllers/Application.scala [hunk 1]
```diff
@@ -13,6 +13,10 @@ import com.gu.recipeasy.models._
 import com.gu.recipeasy.views
 import models._
 import models.CuratedRecipeForm._
+import auth._
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
```
Comment (LATaylor-guardian) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: You can use the play execution context: `play.api.libs.concurrent.Execution.Implicits._`. https://www.playframework.com/documentation/2.5.x/ThreadPools

**Comment by LATaylor-guardian** [comment]
Other than my comment 👍

**Comment by shtukas** [comment]
Added item on the global activity page

screen shot 2017-01-16 at 17 35 59

File: ui/app/com/gu/recipeasy/controllers/Application.scala (hunk 1 above)
Comment (shtukas) [inline ui/app/com/gu/recipeasy/controllers/Application.scala]: Actually I don't need those imports. Cheers,

**Pull request merged**